modulemd/v2/tests/test-modulemd-modulestream.c

```diff
@@ -616,6 +616,54 @@ module_stream_v2_test_depends_on_stream (ModuleStreamFixture *fixture,
   g_clear_object (&stream);
 }
 
+static void
+module_stream_test_community (ModuleStreamFixture *fixture,
+                                         gconstpointer user_data)
+{
+  g_autoptr (ModulemdModuleStream) stream = NULL;
+  guint64 version;
+  g_autofree gchar *community = NULL;
+  
+  for (version = MD_MODULESTREAM_VERSION_ONE;
```

Comment:
You can't just run through both versions here because this isn't a function and attribute provided by the parent class. You need to run tests separately on each of the two `ModuleStream` subclasses.